init-dbt-resources.sh

```diff
@@ -2,9 +2,14 @@
 set -e
 
 psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" --dbname "$POSTGRES_DB" <<-EOSQL
-    CREATE USER IF NOT EXITST $DBT_POSTGRES_USER WITH PASSWORD '$DBT_POSTGRES_PASSWORD';
-    CREATE SCHEMA IF NOT EXISTS $DBT_POSTGRES_SCHEMA AUTHORIZATION $DBT_POSTGRES_USER;
+    DROP ROLE IF EXISTS $DBT_POSTGRES_USER;
+    CREATE USER $DBT_POSTGRES_USER WITH PASSWORD '$DBT_POSTGRES_PASSWORD';
+    CREATE SCHEMA IF NOT EXISTS $POSTGRES_SCHEMA AUTHORIZATION $POSTGRES_USER;
+
+EOSQL
 
+
+psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" --dbname "$POSTGRES_DB" <<-EOSQL
```

Comment:
The GRANT statements below only apply to the objects available when the command is being executed. We would need to alter the default privilleges on the schema

https://www.postgresql.org/docs/current/sql-alterdefaultprivileges.html